modified_cdc_protocol/operation_types/setup_pcr_plate/protocol.rb

```diff
@@ -0,0 +1,344 @@
+# typed: false
+# frozen_string_literal: true
+
+needs 'Standard Libs/PlanParams'
+needs 'Standard Libs/Debug'
+needs 'Standard Libs/Pipettors'
+needs 'Standard Libs/Units'
+needs 'Standard Libs/AssociationManagement'
+needs 'Standard Libs/LabwareNames'
+
+needs 'Collection Management/CollectionActions'
+
+needs 'Microtiter Plates/MicrotiterPlates'
+
+needs 'Microtiter Plates/PlateLayoutGenerator'
+
+needs 'PCR Libs/PCRComposition'
+
+needs 'Modified CDC Protocol/SetupPCRPlateDebug'
+
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+
+PLATE = 'PCR Plate'
+LOT_NUM_KEY = 'Lot Number'
+
+
+class Protocol
+  # Standard Libs
+  include PlanParams
+  include Debug
+  include Pipettors
+  include Units
+  include LabwareNames
+  include AssociationManagement
+
+  include SetupPCRPlateDebug
+
+  include CollectionActions
+
+  include MicrotiterPlates
+
+  include DiagnosticRTqPCRHelper
+
+  RP_COMPOSITION_KEY = 'rp_composition'.to_sym
+  N1_COMPOSITION_KEY = 'n1_composition'.to_sym
+  N2_COMPOSITION_KEY = 'n2_composition'.to_sym
```

Comment:
As N1, N2, and RP are the names of specific primers, they should not be hard coded here. We may decide to use different primers in the future. I am unclear on whether we are adding them or they come as part of the dried master mix. I don't see code where they are added. We should be getting the identity of the primer/probe mix from the sample name in either case.

Reply:
This was the part that I was not a fan of.   We should discuss tomorrow.
All primer probs come in as an `input_array`  and I needed some way to sort them so they would be added into the MicrotiterPlate correctly.   I ended up sorting and putting them in temporary associations.

Reply:
I think you should just rely on the order they are added in the planner. You need to know what order they are in when they go on the thermocycler, but that doesn't have to be hard-coded. Hard-coding becomes really hard to maintain as soon as you start using other primers.